drivers/SmartThings/philips-hue/src/hue/api.lua

```diff
@@ -83,13 +104,13 @@ function PhilipsHueApi.new_bridge_manager(base_url, api_key, socket_builder)
     while true do
       local msg, err = control_rx:receive()
       if err then
-        log.error("Error receiving on control channel for REST API thread", err)
+        log.error_with({ hub_logs = true }, "Error receiving on control channel for REST API thread", err)
         goto continue
       end
 
       if msg and msg._type then
         if msg._type == ControlMessageTypes.Shutdown then
-          log.trace("REST API Control Thread received shutdown message");
+          log.trace_with({ hub_logs = true }, "REST API Control Thread received shutdown message");
```

Comment:
Info level? Maybe other logs happen that indicate this as well, otherwise it seems like a noteworthy event.

Reply:
This is for graceful shutdowns; hard crashes are logged at error.